spec/controllers/api/v1/rents_controller_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'rails_helper'
+
+describe RentsController, type: :controller do
+  describe 'GET #index' do
+    context 'without authenticated user' do
+      subject!(:http_request) { get :index }
+
+      let!(:rents) { create_list(:rent, 5) }
```

Comment:
It's unnecessary create_list. Index method don't execute because fails before by unauthenticated user

Reply:
Done!